src/index.js

```diff
@@ -40,6 +40,53 @@ const _views = new WeakMap();
 
 let bypass = false;
 
+function nextOrPreviousProject (index) {
+    // Generate an array of projects by selecting children of 'project-viewer > ul[is="pv-list-tree"]'. For each 'li[is="pv-list-nested-item"]', expand it into its children.
+    const sidebar = document.getElementsByTagName('project-viewer')[0];
+    let list = sidebar.getElementsByTagName('ul')[0].children;
+
+    var output = [];
+
+    var expandProjects = function (element) {
```

Comment:
I'm not sure what you're talking about? Do you mean how `expandProjects()` is inside `nextOrPreviousProject()`? I did that for the sake of variable scopes, as it's easier to just tack things on to the end of `output` this way.